.ebextensions/deploy.config

```diff
@@ -7,3 +7,11 @@ packages:
 commands:
     01-upgrade-pip:
         command: "/opt/python/run/venv/bin/pip  install --upgrade pip"
+
+files:
+    "/etc/logrotate.elasticbeanstalk.hourly/logrotate.elasticbeanstalk.httpd.conf":
+        mode: "000644"
```

Comment:
What is the default file permissions/owner of this file, do we actually need to configure it?